src/30_Advanced/Embedding_videos_in_amp-carousel.html

```diff
@@ -0,0 +1,140 @@
+<!--
+  #### Introduction
+
+  While it's possible to [integrate videos](https://ampbyexample.com/advanced/integrating_videos_in_amp_an_overview/) in AMP HTML files, when having different videos that you want to display to users, these could be embedded inside `amp-carousel` to offer a better experience.
+-->
```

Comment:
This intro does not really motivate the sample. Have a look at the intro in "Image Galleries with amp-carousel". It clearly states the use case and the challenge. What has the challenge been for this sample?